puppetboard/static/js/environments.js

```diff
@@ -0,0 +1,11 @@
+(function () {
+  var $;
+
+  $ = jQuery;
+
+  $('#switch_env').change(function() {
+    path = location.pathname.split('/');
+    path[1] = $(this).find(':selected').text();
+    location.assign(path.join('/'))
```

Comment:
That's awesome @daenney, I would still like to work on bootstrap to have the added benefit of being mobile friendly. But then again I don't know Semantic UI that well either...